examples/inverse/irl_gail.py

```diff
@@ -97,15 +97,12 @@ def test_gail(args=get_args()):
     # train_envs = gym.make(args.task)
     train_envs = SubprocVectorEnv(
         [lambda: NoRewardEnv(gym.make(args.task)) for _ in range(args.training_num)],
-        norm_obs=True,
     )
+    train_envs = VectorEnvNormObs(train_envs)
     # test_envs = gym.make(args.task)
-    test_envs = SubprocVectorEnv(
-        [lambda: gym.make(args.task) for _ in range(args.test_num)],
-        norm_obs=True,
-        obs_rms=train_envs.obs_rms,
-        update_obs_rms=False,
-    )
+    test_envs = SubprocVectorEnv([lambda: gym.make(args.task) for _ in range(args.test_num)])
+    test_envs = VectorEnvNormObs(test_envs, update_obs_rms=False)
+    test_envs.set_obs_rms(train_envs.get_obs_rms())
```

Comment:
Just mentioning in general: I think the way tianshou is currently dealing with obs_rms and similar things is suboptimal for use cases going beyond research. In an application, I want to train an agent on some env, and to use it on another one. If some running mean is computed on train envs and used for normalization, this mean should be saved as part of the model or env wrappers, and reloaded from a checkpoint together with it. 

I am not sure what the best way is for this, maybe stateful env wrappers that are saved as part of the checkpoint... Not an urgent thing, but if you have a proposal for dealing with this, might be worth mentioning in the tutorials/examples

Reply:
let's leave it as-is since it's beyond this PR's scope